controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -676,24 +688,18 @@ bool wfa_ca::get_send_1905_1_tlv_hex_list(std::list<tlv_hex_t> &tlv_hex_list,
                 return false;
             }
 
-            // Validate hex notation on value or list of values separated by space
+                // Validate hex notation on list of values separated by space
             auto values = string_utils::str_split(it->second, ' ');
-            for (auto value : values) {
+                for (const auto &value : values) {
```

Comment:
What happened to indentation here?

Reply:
I have no idea, but it was fixed in follow-up commit because I don't see it in my code.
This fix requires to rebase with edit, so I will fix it when I'll rebase.